cpp/ql/src/semmle/code/cpp/Type.qll

```diff
@@ -538,7 +611,11 @@ class SignedCharType extends CharType {
 }
 
 /**
- * The C/C++ short types. See 4.3.
+ * The C/C++ short types. See 4.3.  This includes `short`, `signed short`
+ * and `unsigned short`; `short` and `signed short` are equivalent.
```

Comment:
```suggestion
 * and `unsigned short`.
```

I think it's misleading to say they're equivalent when they're actually distinct in QL. They're equivalent in the language, sure, but I don't think that's how people will read this.

Same thing for other integer types.

Reply:
I'll do the others as well...

Reply:
Done.